docs/reference/announcements.md

```diff
@@ -10,6 +10,16 @@ Scheduled release date: 8th of Oct 2024
 
 Scheduled end of maintenance: 14th of April 2026
 
+### Configuring license keys
+
+Camunda 8 components are now able to consume Enterprise licensing information. For additional details, review the [blog post on licensing updates for Camunda 8 Self-Managed](https://camunda.com/blog/2024/04/licensing-update-camunda-8-self-managed/).
+
+Review the documentation for your components for more information on how to provide the license key to each component as an environment variable. To configure with Helm, visit the [Self Managed installation documentation](/self-managed/setup/install.md).
+
+:::note
+Camunda 8 components without a valid license may display **Non-Production License** in the navigation bar and issue warnings in the logs. These warnings have no impact on startup or functionality. To obtain a license, visit the [Camunda Enterprise page](https://camunda.com/platform/camunda-platform-enterprise-contact/).
```

Comment:
Could we amend this line, please? The overall announcement is not 100% correct since Web Modeller has a limitation of 5 users: "These warnings have no impact on startup or functionality."
Maybe be more accurate to link to a https://camunda.com/pricing/ page? We don't have a dedicated license page.